README.md

```diff
@@ -443,7 +443,9 @@ There are multiple ways to run the Cypress tests:
 
 - Run `npx cypress run` to run the tests in the CLI. To have a slightly more interactive
   experience, you can instead run `npx cypress open`. Note: the database,
-  frontend, and backend must be running prior to starting the Cypress tests.
+  frontend, and backend must be running prior to starting the Cypress tests. 
+  You must be connected to the VPN, and the APP_ENV environment variable
```

Comment:
Just as another data point: I was able to run the tests on this branch locally without being on the VPN. Are there particular tests that require VPN access?

Reply:
hmm, let me remove that